students/mia/users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+# Create your models here.
+
+class User(models.Model): 
+    name          = models.CharField(maax_length=50)
+    email         = models.EmailField(max_length =100, unique=True)
+    password      = models.CharField(max_length=50)
+    mobile_number = models.PositiveIntegerField(max_length=50)
+    created_at    = models.DateTimeField(auto_now_add= True)
+    updated_at    = models.DateField(auto_now=True)
```

Comment:
작성 잘해주셔서 감사합니다 예은님! 

다만 `updated_at` column 또한 DateTimeField로 수정해주세요!
추가로 DateField 와 무슨 차이가 있는지도 찾아보시면 좋을 것 같습니다. (그래서 왜 DateTimeField를 쓰는지도요!)

```suggestion
    created_at    = models.DateTimeField(auto_now_add= True)
    updated_at    = models.DateTimeField(auto_now=True)
```